core/modules/widgets/navigator.js

```diff
@@ -527,6 +527,7 @@ NavigatorWidget.prototype.handleImportTiddlersEvent = function(event) {
 	});
 	// Deselect any suppressed tiddlers
 	$tw.utils.each(importData.tiddlers,function(tiddler,title) {
+		// count the keys in the tiddler object
```

Comment:
Unless we intend to add comments everywhere that core util functions are used, why do we need this comment for this specific instance of `$tw.utils.count` being called?

Reply:
Because of this: #5466 "[IDEA] Pls let us rename `$tw.utils.count(tiddler)` to eg: `$tw.utils.cntObjectKeys(xxxx)` "

Reply:
@pmario that is no reason to add a spurious comment for one particular instance of a $tw.util method being called. Either we do it consistently everywhere, or we don't. There is no reason this particular usage of that method is more confusing and in need of clarification than others.